reports-backend/src/main/java/org/jboss/da/reports/impl/ReportsGeneratorImpl.java

```diff
@@ -68,6 +80,16 @@
     }
 
     @Override
+    public Optional<AdvancedArtifactReport> getAdvancedReportFromSCM(SCMLocator scml) throws ScmException,
+            PomAnalysisException, CommunicationException {
+        Optional<ArtifactReport> artifactReport = getReportFromSCM(scml);
+        // TODO: hardcoded to git
+        // hopefully we'll get the cached cloned folder for this repo
+        File repoFolder = scmManager.cloneRepository(SCMType.GIT, scml.getScmUrl(), scml.getRevision());
```

Comment:
I'm not very proud of this :disappointed: , but it seems to work with the suggestion.